packages/devtools_app/lib/src/screens/memory/panes/chart/controller/memory_tracker.dart

```diff
@@ -89,14 +89,23 @@ class MemoryTracker {
     }
   }
 
+  void onIsolateEvent(Event data) {
+    if (data.kind != EventKind.kIsolateExit) {
+      return;
+    }
+    _isolateHeaps.remove(data.isolate!.id);
```

Comment:
nit: this may be easier to follow if it is written as:

```
    if (data.kind == EventKind.kIsolateExit) {
      _isolateHeaps.remove(data.isolate!.id);
    }
```